jdi-light-angular-tests/src/test/java/io/github/epam/angular/unit/tests/with/mocks/CheckboxTest.java

```diff
@@ -0,0 +1,80 @@
+package io.github.epam.angular.unit.tests.with.mocks;
+
+import com.epam.jdi.angular.elements.common.Checkbox;
+import com.epam.jdi.light.elements.common.UIElement;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static junit.framework.Assert.assertFalse;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertTrue;
+
+public class CheckboxTest {
```

Comment:
Move mock tests to separate module
Write real tests for checkboxes instead of mocks

Reply:
Removed these tests for now, I'll add them to a new module in the next PR. 
Also, created #2097 for the missing unit tests.